catalog/locallibrary/settings.py

```diff
@@ -24,7 +24,7 @@
 import os
 SECRET_KEY = os.environ.get('DJANGO_SECRET_KEY', 'cg#p$g+j9tax!#a3cup@1$8obt2_+&k3q+pmu)5%asj6yjpkag')
 
-DATABASE_HOST = os.environ.get('POSTGRES_HOST', '')
+DATABASE_HOST = os.environ.get('POSTGRES_CATALOG_HOST', '')
```

Comment:
I'd take another approach to this.
I would use the POSTGRES_HOST for both the recomendator and the catalog, but pass different .env files to each container, each with its own values.

Reply:
yep, makes sense. The environment variable can have the same name across environments